content/docs/getting-started.md

```diff
@@ -28,12 +28,19 @@ to get started, enter:
   ipfs cat /ipfs/QmYwAPJzv5CZsnA625s3Xf2nemtYgPpHdWEz79ojWnPbdG/readme
 
 ```
+<div class="alert alert-warning">
+If you're running IPFS in the cloud, you should initialize IPFS with the `server` profile (`ipfs init --profile=server`) to avoid aggravating your service provider. See [go-ipfs#4343](https://github.com/ipfs/go-ipfs/issues/4343) for details.
+</div>
+
+<div class="alert alert-info">
+The hash after `peer identity: ` is your peer's ID. That's how other nodes on the network can find and connect to you. Don't bother writing it down, you can run `ipfs id` at any time if you need it.
+</div>
 
 <div class="alert alert-info">
-Note the hash there may differ. If it does, use the one you got.
+Note, `HASH` in the `ipfs cat /ipfs/HASH/readme` command may differ. If it does, use the one you got. Do *not* use your peer ID.
```

Comment:
> Do *not* use your peer ID.

This sounds really ominous, like you'll break something if you use your peer ID. I think you just want to say something like "Don't confuse this with the peer identity hash -- the peer identity hash is something your IPFS node uses to identify itself to other nodes. You want the content hash from the line that looks like `ipfs cat /ipfs/QmYwAPJzv5CZsnA625s3Xf2nemtYgPpHdWEz79ojWnPbdG/readme`

Reply:
Speaking of which, we need a HAL easter egg.

Reply:
Fixed. Thoughts? I'm trying to keep the peer ID explanation as a separate note.